### PR TITLE
Adds Elyran Standard language

### PR DESCRIPTION
**Pull request by Jaraci**
Uh oh. The big one.

- Implements Elyran Standard as the new Elyran language to accompany the Elyra rework.
- Elyran standard now occupies the ,4 language code. Sign language has been moved to ,s to accommodate this.

image
image

To consider:

- adjusting the space_chance to make words a little less long?
- adjusting the colour? Yellow on white is hard
- figuring out what else I missed
- how to word my changelog

### CODE REVIEW

**Comment by Jaraci** [comment]
!review

**Comment by FayeLovesU** [comment]
What does it look like if you can't understand Elyran Standard?

**Comment by Caelphon** [comment]
Should it need Loremaster approval, it does have it 👍

**Comment by TheDanseMacabre** [comment]
Pan-Slavic next??

File: code/modules/mob/language/generic.dm [hunk 1]
```diff
@@ -80,7 +80,7 @@
 	signlang_verb = list("signs", "gestures")
 	sing_verb = null
 	colour = "i"
-	key = "4"
+	key = "s"
```
Comment (NonQueueingMatt) [inline code/modules/mob/language/generic.dm:83]: Remember to put this in the changelog.

**Comment by Jaraci** [comment]
> What does it look like if you can't understand Elyran Standard?

image

Sol Common shown as a basis for comparison. The language-not-understood code itself is far from perfect, but I think it's distinguished enough from the other human languages when scrambled.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]
Test Elyran is my favourite character

**Pull request merged**